gems/sorbet/test/snapshot/run_one.sh

```diff
@@ -164,17 +164,26 @@ fi
   bundle config set no_prune true
   bundle install --verbose --local
 
-  info "├─ Checking installation"
+  info "├─ Checking srb commands"
   bundle check
 
+  # srb commands to run are stored in the test under `src/test.sh`
+  test_script="test.sh"
+  if ! [ -f "$test_script" ]; then
+    error "├─ No $test_script file in $(pwd)"
+    error "└─ Can't run srb"
+    exit 1
+  fi
+  chmod +x "$test_script"
```

Comment:
What do you think about having this fall back on setting "$test_script" to the original command when the file doesn't exist? That way we could include `test.sh` only when it's absolutely necessary.

Reply:
Done. Also updated the readme 👍